_data/packages.yml

```diff
@@ -22,7 +61,7 @@
       name: Hans Moritz Günther
       github_username: hamogu
   eic:
-      name: Chiara Marmo
+      name:
```

Comment:
Something odd is happening here as well where Chiara's name is being erased. why is this happening? her name is on her github profile so something is happening here in pyosmeta.

Reply:
Oh, this is interesting. eic is a new field! Well newish, we never updated the workflow to handle it

I did something here - https://github.com/pyOpenSci/pyosMeta/issues/141 so it's supported in the model but it's not a valid contrib type! so this is a  real pyosmeta bug!! Let's fix this asap! 🚀 

https://github.com/pyOpenSci/pyosMeta/issues/233

Reply:
https://github.com/pyOpenSci/pyosMeta/pull/235

ok this is fixed. there were a few issues with how eic was setup and checked in our workflow. So i updated the model, and the field_serializer  and finally our cli script to ensure eic field was properly checked. this should be fixed now. 

i wonder if we want a second ci build that we can run that runs from the main branch vs pypi? we could release a patch now to see how many things are fixed. or we could be patient and just consider a separate Ci build. 

Or we could setup a CI build in the pyosmeta repo too?